### DISCUSSION TITLE
SumatraPDF pre-release sending and receiving data to IP 216.24.57.1

### DISCUSSION THREAD

**Discussion by OAStubby** [sumatrapdfreader/sumatrapdf#2539] [Q&A]
Why is it that every time I open an epub with SumatraPDF pre-release build version 14547 there is a sending and receiving of data to IP address 216.24.57.1 (render.com)?

**Comment by kjk** [comment]
When a crash happens we send crash report to updatecheck.io which is hosted on render.com
See https://github.com/sumatrapdfreader/sumatrapdf/blob/master/src/CrashHandler.cpp#L30

It also happens when program encounters a condition that I think is a bug and want to gather more info about what happens, so that I can fix the issue. See `ReportIf` in the code.

Can you send me one of such epub files that trigger this so that I can investigate and fix the problem? See https://www.sumatrapdfreader.org/docs/How-to-submit-bug-reports

**Comment by kjk** [comment]
Related bug report: https://github.com/sumatrapdfreader/sumatrapdf/issues/2540
